metadata.md

```diff
@@ -4,6 +4,7 @@ There are eight tables that are described below. example data is stored in [data
 
 -   [Measurement](#Measurement) ([Measurements.csv](data/Measurements.csv))
 -   [Sample](#Sample) ([Sample.csv](data/Sample.csv))
+-   [ContextualMeasurement](#ContextualMeasurement) ([ContextualMeasurement](data/ContextualMeasurement.csv))
```

Comment:
It is so difficult to have a good table name. The `contextual measurement` isn't very descriptive (same for other names). 

@hswerdfe and I had thought about changing `Measurement` to `Virus` and then creating new table for the types of measures in `ContextualMeasurment`. Many measures are about wastewater and we where thinking of something like `Wastewater`, which would encompass wastewater and other environmental and "contextual" measures. 

thoughts about an alternative or better name?

Reply:
Not so much of an issue for me. Go ahead if you want to make some name changes.